### ISSUE TITLE
[css] Tables render without padding 

### ISSUE THREAD

**Issue by bong0** [victorstanciu/Wikitten#67]
As in this screenshot from a wiki running the master.zip from 8f32c0b5bedb1e6aeb93cbc2226d8781d3f3ab5d the table css leads to bad/unreadable rendering of table cells. 
![table source and rendered html](https://cloud.githubusercontent.com/assets/179111/9036182/4bd6452a-39df-11e5-840b-dad7a6bcb748.png)

What would be the effort to change this to something like in [mdwiki](http://dynalon.github.io/mdwiki/#!quickstart.md)?

**Comment by asbjornu** [comment]
I assume what's missing is `class="table"` on the `<table>` element.

**Comment by praveenscience** [comment]
I am not sure if we can make Markdown to add extra classes, but we can redefine `table {}` in CSS right. :)

**Comment by asbjornu** [comment]
If we control the Markdown to HTML conversion pipeline or can extend it somehow, we can add `class="table"` to all `<table>` elements, no? That's what would be required to have the Bootstrap style added to it.

**Comment by praveenscience** [comment]
Won't that be against the implementation of original markdown parser? Also also I believe you can add extra features with GFM. I am gonna implement that soon.

**Comment by bong0** [comment]
FYI: I'm not evaluating wikitten anymore but switched to a dokuwiki based solution. It has pretty nice markdown support and also one fairly good bootstrap theme.

**Comment by asbjornu** [comment]
@praveenscience The parser shouldn't be involved in this at all. Adding `class="table"` would be done in the rendering of whatever the parser creates (an [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree), I assume) from the Markdown markup. If it's possible to add a plugin to the rendering pipeline, that would be an elegant and simple solution.

**Comment by praveenscience** [comment]
@asbjornu Agreed. There's a plan of using Parsedown instead of the original markdown, so this opens up to so many possibilities. I will try to check with the PD guys, if there's a possibility of adding extra class. :)
